src/mpid/ch4/netmod/ucx/ucx_init.h

```diff
@@ -216,6 +216,7 @@ static inline int MPIDI_NM_mpi_init_hook(int rank,
 
 #ifndef HAVE_DEBUGGER_SUPPORT
     MPIDI_UCX_global.lw_send_req = MPIR_Request_create(MPIR_REQUEST_KIND__SEND);
+    // TODO: is it safe to assume a request will always be allocated
```

Comment:
We should abort if this request is not allocated successfully during init.

Reply:
Agree.